tests/framework/microvm.py

```diff
@@ -842,23 +843,30 @@ def ssh(self):
 class MicroVMFactory:
     """MicroVM factory"""
 
-    def __init__(self, base_path: Path, fc_binary_path: Path, jailer_binary_path: Path):
+    def __init__(
+        self, base_path: Path, fc_binary_path: Path, jailer_binary_path: Path, **kwargs
+    ):
         self.base_path = Path(base_path)
         self.vms = []
         self.fc_binary_path = Path(fc_binary_path)
         self.jailer_binary_path = Path(jailer_binary_path)
+        self.kwargs = kwargs
 
     def build(self, kernel=None, rootfs=None, **kwargs):
         """Build a microvm"""
+        kwargs = self.kwargs | kwargs
+        microvm_id = kwargs.pop("microvm_id", str(uuid.uuid4()))
```

Comment:
what happens if `microvm_id` is not a key in `kwargs`?

Reply:
The 2nd parameter acts as default, `str(uuid.uuid4())`

Reply:
cool! I didn't know that